python/examples/mlir/memref_management.py

```diff
@@ -0,0 +1,117 @@
+import torch
+import ctypes
+
+from mlir import ir
+from mlir.dialects import func, memref
+from mlir.runtime import np_to_memref
+from mlir.execution_engine import ExecutionEngine
+from mlir.passmanager import PassManager
+
+import lighthouse.utils as lh_utils
+
+
+def create_mlir_module(ctx: ir.Context, shape: list[int]) -> ir.Module:
```

Comment:
```suggestion
def create_mlir_module(shape: list[int]) -> ir.Module:
```
See below for how context can be elided.